### PR TITLE
Carol/docs/proctored exams

### PR DESCRIPTION
**Pull request by mhoeber**
(This is from @catong )

New PR integrates proctored exam documentation with academic course credit doc.
This PR supercedes https://github.com/edx/edx-documentation/pull/451for course author doc and #458 for student facing doc.

DRAFT course author doc is here: 
http://draft-course-author-proctored-exams.readthedocs.org/en/latest/building_course/credit_courses/proctored_exams.html

DRAFT student facing doc is here:
http://draft-student-proctored-exams.readthedocs.org/en/latest/SFD_credit_courses/SFD_proctored_exams.html

Please review for technical accuracy and alignment with Product vision.
@chrisndodge @griffresch 
FYI @srpearce @jhendersonedx @mhoeber

### CODE REVIEW

**Comment by mhoeber** [comment]
@catong

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 1]
```diff
@@ -1,7 +1,647 @@
...
+might be evidence of cheating.
+
+In for-credit courses, course teams can create exams that they designate as
+proctored exams, and make these exams requirements for credit eligibility. 
+
+.. note:: Only learners enrolled in the Verified track see the option to take
+   an exam with online proctoring. If you believe that there are students who
+   have not yet upgraded to the Verified track who should be taking proctored
+   exams and qualifying for course credit, you should remind them to enroll in
+   the Verified track before the part of your course where proctored exams
```
Comment (griffresch) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @catong  Should say something like "...remind them to enroll in the Verified track before the deadline to upgrade has passed."

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 2]
```diff
@@ -1,7 +1,647 @@
...
+
+When proctoring session results become available, course staff can download a
+report that lists proctoring results for learners in their course. For more
+information, see :ref:`Proctored Exam Results`.
+
+.. For passing students, is there some visible status change for certificates
+.. etc. that is triggered to indicate that they can go ahead with requesting
+.. credit? Anything visible in the UI?
+
+Learners who have achieved the minimum credit-eligible grade in the course as
```
Comment (griffresch) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @catong the process for Credit eligibility is: user must successfully complete ICRV blocks in the course, successfully pass the proctoring part of the exams in the course, and reach the minimum credit eligibility passing grade. For just proctoring, if a user gets a Satisfactory result for the proctoring session, the eligibility table on the Progress page reflects that. The minimum passing score is a separate line item that is calculated from the overall course score, not just from the score from the exam with proctoring. 

I'd recommend taking out the first part of this line "Learners who have achieved he minimum credit-eligible grade in the course..." Instead, just talk about proctoring and link to the Credit Eligibility section of the Credit docs.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 3]
```diff
@@ -1,7 +1,647 @@
...
+.. _CA_LearnerExperience_Proctored Exams:
+
+==============================================
+The Learner Experience of Proctored Exams
+==============================================
+
+Only learners who have signed up for the Verified track in your course have
+the option of taking proctored exams. When they access a proctored exam,
+learners on the Verified track can choose to take the exam with online
+proctoring, or to take the exam without proctoring and only as a timed exam.
```
Comment (griffresch) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @catong remove the last part "and only as a timed exam." When a user decides to take the exam as "Open" there are no timing or proctoring elements on it.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 4]
```diff
@@ -1,7 +1,647 @@
...
+
+==========================================
+Running Out Of Time In an Exam
+==========================================
+
+If the timer reaches 00.00 before a learner has completed the exam, all the
+answers that the learner has submitted up to that point in time are submitted
+for grading.
+
+If the exam is also a proctored exam, the proctoring session automatically
```
Comment (griffresch) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @catong  this is a bit confusing. All proctored exams are timed.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst (hunk 1 above)
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @griffresch, @jhendersonedx, In some cases, might proctored exams occur in the course well before the deadline to upgrade? The reason I phrased the reminder this way is to address that issue. If students are not yet on the verified track, they won't be exposed to the in-course reverification steps nor will they even see proctored exams. Please let me know what's best to say here.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst (hunk 3 above)
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: Done

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst (hunk 4 above)
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: Fixed; these references to timed exams are from an earlier time when I thought we were also releasing the ability to create timed exams without proctoring, and that exams taken without proctoring were timed exams.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst (hunk 2 above)
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: Rewritten

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 5]
```diff
@@ -1,7 +1,647 @@
...
+
+If learners choose to take the exam without proctoring, they are not eligible
+for credit for the exam, and will not be offered the proctored option for that
+exam again.
+
+Learners who are not ready to make the choice or take the exam immediately can
+defer their decision and return to the exam again at some other time before
+the due date.
+
+.. Do we want to give advice to course staff about planning an adequate due date?
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst]: @griffresch  or @jhendersonedx ?

File: en_us/course_authors/source/building_course/credit_courses/online_proctoring_rules_students.rst [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+EdX.org uses an online proctoring service for certain exams (typically final
+exams) that are part of courses that can be made eligible for credit. Learners
+taking proctored exams are required to adhere to a set of defined rules to
+receive a **Satisfactory** result from the proctoring service.
+
+If a particular exam in your course has less stringent rules or allows you to
+use a specific tool such as a calculator or reference book during a proctored
+exam, your course staff will let you know. If you are not told about any
+different proctoring exam rules, make sure you adhere to the "Closed Book
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/online_proctoring_rules_students.rst]: @chrisndodge I modified these introductory paragraphs in the Proctoring Rules to address your comment about "Closed Book" exam rules. OK?

File: en_us/course_authors/source/building_course/credit_courses/online_proctoring_rules_students.rst [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+Exam" rules that are described in the following sections.
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/online_proctoring_rules_students.rst:16]: @chrisndodge Added clarification re "Closed Book" rules here and elsewhere.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 8]
```diff
@@ -1,7 +1,756 @@
...
+For more information about the technical requirements for taking a proctored
+exam, and edX's :ref:`Online Proctoring Rules <Online Proctoring Rules>`, see
+:ref:`Preparing Learners for Proctored Exams`.
+
+.. note:: Course staff should familiarize themselves with edX's :ref:`Online
+   Proctoring Rules <Online Proctoring Rules>`, which reflect our proctoring
+   software partner's "Closed Book Exam" rules. These strict rules prohibit
+   learners from using any tools during the exam, including pencil and
+   paper,calculators, or reference books.
+
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst:76]: @chrisndodge Added note to clarify that our published rules are the "Closed Book" rules

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 9]
```diff
@@ -1,7 +1,756 @@
...
+===================================
+
+In some situations, for example to accommodate learners with disabilities,
+additional time allowances can be provided for specific students. Consult with
+your organization's Disability Services resources to decide whether and how a
+learner with specific needs can be accommodated for a timed exam.
+
+.. note:: Make sure the learner who has requested additional time does not
+   start taking the proctored exam before the additional time has been
+   approved and set up.
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst:406]: @chrisndodge Added note re ensure learner does not start exam before allowance has been granted and set up.

File: en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst [hunk 10]
```diff
@@ -1,7 +1,756 @@
...
+=======================================
+
+In some situations, for example to accommodate learners with disabilities,
+special allowances can be provided for specific students. Consult with your
+organization's Disability Services resources to decide whether and how a
+learner with specific needs can be accommodated.
+
+.. note:: Make sure the learner who has requested special allowances does not
+   start taking the proctored exam before the exemption has been approved and
+   set up.
```
Comment (catong) [inline en_us/course_authors/source/building_course/credit_courses/proctored_exams.rst:437]: @chrisndodge Added note re ensure learner does not start exam before allowance has been granted and set up.

**Comment by catong** [comment]
Closing this PR; will open new PR that merges to master rather than to Sylvia's credit eligibility branch.

**Pull request closed without merging**